coalib/output/ConsoleInteraction.py

```diff
@@ -609,6 +609,51 @@ def print_actions(console_printer, section, actions, failed_actions):
     return get_action_info(section, actions[choice - 1], failed_actions)
 
 
+def apply_action_to_section(action,
+                            section,
+                            action_name,
+                            result,
+                            file_dict,
+                            file_diff_dict,
+                            console_printer,
+                            failed_actions):
+    """
+    Applies the given action.
+
+    :param action:          The action to the applied.
+    :param section:         Currently active section.
+    :param action_name:     The name of the action.
```

Comment:
Yeah, actually the `ask_for_action_and_apply` already has `action_name` defined via the `print_action` function ( this [line](https://github.com/coala/coala/pull/4112/files/ebef35bd716b9bdb26e0a64543509d9e1ab1e1a3..0e2b15b19ed0fb39d4242c517611b617df7d573f#diff-5be6f2c646fedbab1fd2aea142f54303R686)), so I thought just to propagate it through.